### DISCUSSION TITLE
Commits appearing to be doubled in dashboard, but not in repos themselves

### DISCUSSION THREAD

**Discussion by wjrm500** [community/community#41065] [New to GitHub]
Commits appear to be counted twice on my user dashboard, see images for reference:

![image](https://user-images.githubusercontent.com/48405095/206380588-53787f74-c816-4532-b6ed-0b3490335c03.png)
![image](https://user-images.githubusercontent.com/48405095/206380677-25b09a40-dca1-42f4-aa43-dba13232665c.png)

What can I do to prevent the doubling?

**Comment by queenofcorgis** [comment]
Hmmm do you have multiple emails connected to your account/commits?
